crates/viewer/re_ui/src/command.rs

```diff
@@ -346,6 +353,8 @@ impl UICommand {
 
             #[cfg(target_arch = "wasm32 ")]
             Self::ViewportMode(_) => None,
+
+            Self::ToggleChunkBasedDataDensityGraph => Some(ctrl_shift(Key::D)),
```

Comment:
```suggestion
            Self::ToggleChunkBasedDataDensityGraph => Some(cmd_shift(Key::D)),
```

more mac friendly :)